src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -145,11 +144,78 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
     if (cartState === "FETCHING_QUOTA") {
       const message =
         error.message ?? "Encounted an error while fetching quota";
```

Comment:
how about adding `Encounted an error while fetching quota` to ERROR_MESSAGE too haha

Reply:
missed this out, thanks!! added!